Sdtrig.tex

```diff
@@ -86,13 +86,19 @@ \section{Actions}
 \hline
 5 & Reserved for use by the trace specification. \\
 \hline
-8 -- 9 & Signal the firing of the trigger to other blocks within the hart (e.g. as countable events to hpmcounters).  Use external debug trigger output 0 or 1 (respectively). \\
+8 -- 9 & Send a signal to TM external trigger output 0 or 1 (respectively).  the
+firing of the trigger to blocks. \\
```

Comment:
This second sentence looks like an editing error.

Reply:
Thanks.